wasmer/wasmer.h

```diff
@@ -1092,6 +1092,11 @@ uint64_t wasmer_instance_get_runtime_breakpoint_value(wasmer_instance_t *instanc
  */
 bool wasmer_instance_is_function_imported(wasmer_instance_t *instance, const char *name);
 
+/**
+ * Reset an WebAssembly instance, cleaning memories and globals
+ */
+wasmer_result_t wasmer_instance_reset(wasmer_instance_t *instance);
```

Comment:
is this intended?

Reply:
It's a copy of the newest `wasmer.h`, yes/